src/components/atoms/Button.module.css

```diff
@@ -66,6 +67,23 @@
   background: var(--brand-white);
 }
 
+.outline {
+  background: linear-gradient(
+    to right,
+    var(--color-primary) 50%,
+    var(--brand-white) 50%
+  );
+  background-size: 202% 100%;
```

Comment:
question(non-blocking): I get suspicious when I read magic numbers like 202. Can we try to work with multiples of a base value?

Reply:
200% was working fine on Chrome but Firefox for some reason was showing a little white space at the end, not sure how to fix the problem differently. I could put a bigger round number (like 300%) but I don't think it would make it clearer and that would also affect the transition speed, would this option be better in your opinion?

Reply:
Thanks for the clarification. Adding a comment explaining that this is a firefox specific fix should be sufficient then.